instagram/src/components/PostContainer.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import Styled from 'styled-components'
+
+import CommentSection from './CommentSection.js'
+
+
+function PostContainer(props) {
+    const post = props.postData[0];
+    console.log(post);
+    return (
+
+        <div className="postContainer">
+
+            {props.postData.map(post => {
```

Comment:
like I posted in chat, we wanna map over our data in <app/> and make a post container for _each_ post, just a quick refactor!